docs/quickstart.md

```diff
@@ -0,0 +1,177 @@
+# Quickstart with firecracker-containerd
+
+This quickstart guide provides simple steps to get a working
+firecracker-containerd environment, with each of the major components built from
+source.  Once you have completed this quickstart, you should be able to run and
+develop firecracker-containerd (the components in this repository), the
+Firecracker VMM, and containerd.
+
+This quickstart will clone repositories under your `$HOME` directory and install
+files into `/usr/local/bin`.
+
+1. Get an AWS account
+2. Launch an i3.metal instance running Amazon Linux 2 (you can find it in the
+   EC2 console Quickstart wizard, or by running
+   `aws ssm get-parameters --names /aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2`
```

Comment:
Would it be worth, for clarity, to give an ami-id in a specific region that we know works with this guide and add the notes about finding others for other regions?

Reply:
I'm leery of putting an AMI ID here because I don't know that we'll be able to update it frequently.

Reply:
Why would it need to be updated frequently if everything is versioned?

The flip side is are we going to test/verify that it works with the latest AL2 ami going forward?

Reply:
The Amazon Linux team releases new AMIs when there are certain classes of security updates.  I'd prefer not to unintentionally recommend an insecure AMI ID and instead point to the latest recommended AMI ID (which is discoverable via the SSM parameters).

Reply:
SGTM. I hadn't really thought about the security part of it.